simpl/threadlocal.py

```diff
@@ -0,0 +1,52 @@
+"""Threadlocal."""
+import collections
+import threading
+
+THREAD_STORE = threading.local()
+DEFAULT_NAMESPACE = 'call_context'
+
+
+class ThreadLocalDict(collections.MutableMapping):
+
+    """A dict whose data is local to the thread."""
+
+    def __init__(self, varname, *args, **kwargs):
+        self.varname = varname
```

Comment:
Statically putting the class name in the repr string would display an incorrect class name when subclassed. `return '<%s %s>' % (self.__class__.__name__, under)` would work even if subclassed.